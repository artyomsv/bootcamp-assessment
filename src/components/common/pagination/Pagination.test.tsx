import * as React from 'react';
import {createShallow} from '@material-ui/core/test-utils';
import Pagination from './Pagination.widget';

const shallow = createShallow();

describe('renders without crashing', () => {
  test('if renders', () => {
    const mockFn = jest.fn();
    const result = shallow(<Pagination onChangePage={mockFn} page={{page: 0, totalPages: 10, totalResults: 200}}/>);
    expect(result).toMatchSnapshot();
  });
});
