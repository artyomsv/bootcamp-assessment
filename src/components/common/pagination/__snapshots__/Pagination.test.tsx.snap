// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing if renders 1`] = `
<Pagination
  classes={
    Object {
      "pagination": "Pagination-pagination-2",
      "root": "Pagination-root-1",
    }
  }
  onChangePage={[MockFunction]}
  page={
    Object {
      "page": 0,
      "totalPages": 10,
      "totalResults": 200,
    }
  }
/>
`;
